lib/index.js

```diff
@@ -101,3 +109,50 @@ exports.defaultDocumentLoader = defaultDocumentLoader;
 
 // expose for testing purposes
 exports._caches = require('./caches');
+
+/**
+ * Takes in a request to a kms system from a proxy and validates it.
```

Comment:
bedrock-kms doesn't know anything about a "proxy". This should just mention that it's doing some kind of network-based authz validation.